custom_components/truenas/manifest.json

```diff
@@ -1,18 +1,14 @@
 {
-  "codeowners": [
-    "@sdwilsh"
-  ],
+  "codeowners": ["@sdwilsh"],
   "config_flow": true,
   "dependencies": [],
   "documentation": "https://github.com/sdwilsh/hass-truenas",
   "domain": "truenas",
   "homekit": {},
   "iot_class": "local_polling",
   "name": "TrueNAS",
-  "requirements": [
-    "aiotruenas-client==0.7.0"
-  ],
+  "requirements": ["aiotruenas-client==0.8.0"],
   "ssdp": [],
-  "version": "0.3.0a0",
+  "version": "0.4.0a0",
```

Comment:
You shouldn't edit this by hand.  When I do a release, I have a script that updates this automatically.